### ISSUE TITLE
Traceback in console when browsing inside Web View

### ISSUE THREAD

**Issue by voronianski** [trueinteractions/tint2#69]
My console is populated with a very long tracebacks when I start browsing inside web view for a while, for example:

```bash
[00:03:28.458] <<< FigDiskCacheRepository >>> fdcr_GetFileStats: stat failed for /var/folders/5z/pgm_8b253pq6nf3rg2kjlfxw0000gn/T//MediaCache/CachedMedia-16X9lZ, err = 2
[00:03:29.077] FigLimitedDiskCacheProvider_CopyProperty signalled err=-12784 (kFigBaseObjectError_PropertyNotFound) (no such property) at /SourceCache/CoreMedia/CoreMedia-1562.19/Prototypes/FigByteStreamPrototypes/FigLimitedDiskCacheProvider.c line 947
[00:03:29.077] FigLimitedDiskCacheProvider_CopyProperty signalled err=-12784 (kFigBaseObjectError_PropertyNotFound) (no such property) at /SourceCache/CoreMedia/CoreMedia-1562.19/Prototypes/FigByteStreamPrototypes/FigLimitedDiskCacheProvider.c line 947
[00:03:29.080] FigLimitedDiskCacheProvider_CopyProperty signalled err=-12784 (kFigBaseObjectError_PropertyNotFound) (no such property) at /SourceCache/CoreMedia/CoreMedia-1562.19/Prototypes/FigByteStreamPrototypes/FigLimitedDiskCacheProvider.c line 947
Dec 15 00:03:29 MacBook-Pro-Dmitri.local rtcreporting[2271] <Info>: logging starts...
Dec 15 00:03:29 MacBook-Pro-Dmitri.local rtcreporting[2271] <Debug>: setMessageLoggingBlock: called
Dec 15 00:03:29 MacBook-Pro-Dmitri.local rtcreporting[2271] <Debug>: startConfigurationWithCompletionHandler: Cached 0 enabled backends
Dec 15 00:03:29 MacBook-Pro-Dmitri.local rtcreporting[2271] <Debug>: setUserInfoDict: enabled backends: (
	)
Dec 15 00:03:29 MacBook-Pro-Dmitri.local rtcreporting[2271] <Debug>: setMessageLoggingBlock: called
Dec 15 00:03:29 MacBook-Pro-Dmitri.local rtcreporting[2271] <Debug>: startConfigurationWithCompletionHandler: Cached 0 enabled backends
Dec 15 00:03:29 MacBook-Pro-Dmitri.local rtcreporting[2271] <Debug>: setUserInfoDict: enabled backends: (
	)
Dec 15 00:03:29 MacBook-Pro-Dmitri.local rtcreporting[2271] <Debug>: setMessageLoggingBlock: called
Dec 15 00:03:29 MacBook-Pro-Dmitri.local rtcreporting[2271] <Debug>: startConfigurationWithCompletionHandler: Cached 0 enabled backends
Dec 15 00:03:29 MacBook-Pro-Dmitri.local rtcreporting[2271] <Debug>: setUserInfoDict: enabled backends: (
	)
[00:03:29.856] <<< FigDiskCacheRepository >>> FigDiskCacheRepositoryReserveBacking: [0x7fb68b7c01f0] id 'https://ssl.gstatic.com/chat/sounds/incoming_video_short_5abdd7c1c7fa8bbd5d6b4733de315c59.mp3' not in checked-out list
[00:03:29.856] crabsSwipeBacking signalled err=-12876 (kFigByteStreamError_AllocationFailed) (no freeable disk blocks) at /SourceCache/CoreMedia/CoreMedia-1562.19/Prototypes/FigByteStreamPrototypes/FigCachedReadAheadByteStream.c line 3942
[00:03:29.857] <<< FigDiskCacheRepository >>> FigDiskCacheRepositoryReserveBacking: [0x7fb68b7c01f0] id 'https://ssl.gstatic.com/chat/sounds/incoming_video_short_5abdd7c1c7fa8bbd5d6b4733de315c59.mp3' not in checked-out list
[00:03:29.857] crabsSwipeBacking signalled err=-12876 (kFigByteStreamError_AllocationFailed) (no freeable disk blocks) at /SourceCache/CoreMedia/CoreMedia-1562.19/Prototypes/FigByteStreamPrototypes/FigCachedReadAheadByteStream.c line 3942
[00:03:29.857] <<< FigDiskCacheRepository >>> FigDiskCacheRepositoryReserveBacking: [0x7fb68b7c01f0] id 'https://ssl.gstatic.com/chat/sounds/incoming_video_short_5abdd7c1c7fa8bbd5d6b4733de315c59.mp3' not in checked-out list
[00:03:29.864] <<< FigDiskCacheRepository >>> FigDiskCacheRepositoryReserveBacking: [0x7fb68b7c01f0] id 'https://ssl.gstatic.com/chat/sounds/incoming_video_short_5abdd7c1c7fa8bbd5d6b4733de315c59.mp3' not in checked-out list
[00:03:29.864] crabsSwipeBacking signalled err=-12876 (kFigByteStreamError_AllocationFailed) (no freeable disk blocks) at /SourceCache/CoreMedia/CoreMedia-1562.19/Prototypes/FigByteStreamPrototypes/FigCachedReadAheadByteStream.c line 3942
[00:03:29.884] <<< FigDiskCacheRepository >>> FigDiskCacheRepositoryReserveBacking: [0x7fb68b7c01f0] id 'https://ssl.gstatic.com/chat/sounds/incoming_video_short_5abdd7c1c7fa8bbd5d6b4733de315c59.mp3' not in checked-out list
[00:03:29.884] crabsSwipeBacking signalled err=-12876 (kFigByteStreamError_AllocationFailed) (no freeable disk blocks) at /SourceCache/CoreMedia/CoreMedia-1562.19/Prototypes/FigByteStreamPrototypes/FigCachedReadAheadByteStream.c line 3942
[00:03:29.884] <<< FigDiskCacheRepository >>> FigDiskCacheRepositoryReserveBacking: [0x7fb68b7c01f0] id 'https://ssl.gstatic.com/chat/sounds/incoming_video_short_5abdd7c1c7fa8bbd5d6b4733de315c59.mp3' not in checked-out list
[00:03:29.891] <<< FigDiskCacheRepository >>> FigDiskCacheRepositoryReserveBacking: [0x7fb68b7c01f0] id 'https://ssl.gstatic.com/chat/sounds/incoming_video_short_5abdd7c1c7fa8bbd5d6b4733de315c59.mp3' not in checked-out list
[00:03:29.891] crabsSwipeBacking signalled err=-12876 (kFigByteStreamError_AllocationFailed) (no freeable disk blocks) at /SourceCache/CoreMedia/CoreMedia-1562.19/Prototypes/FigByteStreamPrototypes/FigCachedReadAheadByteStream.c line 3942
[00:03:30.011] <<< FigDiskCacheRepository >>> FigDiskCacheRepositoryReserveBacking: [0x7fb68b7c01f0] id 'https://ssl.gstatic.com/chat/sounds/incoming_video_short_5abdd7c1c7fa8bbd5d6b4733de315c59.mp3' not in checked-out list
[00:03:30.011] crabsSwipeBacking signalled err=-12876 (kFigByteStreamError_AllocationFailed) (no freeable disk blocks) at /SourceCache/CoreMedia/CoreMedia-1562.19/Prototypes/FigByteStreamPrototypes/FigCachedReadAheadByteStream.c line 3942
[00:03:30.011] <<< FigDiskCacheRepository >>> FigDiskCacheRepositoryReserveBacking: [0x7fb68b7c01f0] id 'https://ssl.gstatic.com/chat/sounds/incoming_video_short_5abdd7c1c7fa8bbd5d6b4733de315c59.mp3' not in checked-out list
[00:03:30.012] <<< FigDiskCacheRepository >>> FigDiskCacheRepositoryReserveBacking: [0x7fb68b7c01f0] id 'https://ssl.gstatic.com/chat/sounds/incoming_video_short_5abdd7c1c7fa8bbd5d6b4733de315c59.mp3' not in checked-out list
[00:03:30.012] crabsSwipeBacking signalled err=-12876 (kFigByteStreamError_AllocationFailed) (no freeable disk blocks) at /SourceCache/CoreMedia/CoreMedia-1562.19/Prototypes/FigByteStreamPrototypes/FigCachedReadAheadByteStream.c line 3942
Dec 15 00:03:30 MacBook-Pro-Dmitri.local rtcreporting[2271] <Debug>: sendMessageWithCategory: (null)
Dec 15 00:03:30 MacBook-Pro-Dmitri.local rtcreporting[2271] <Debug>: sendMessageWithCategory: (null)
Dec 15 00:03:30 MacBook-Pro-Dmitri.local rtcreporting[2271] <Debug>: sendMessageWithCategory: (null)
```

Test application that I use is - https://github.com/voronianski/tint-browser-demo/blob/master/app.js

**Comment by trevorlinton** [comment]
Side note, i've filed a bug with Apple on this as there's no way to remove these trace logs (with any known public API). I'm leaving this as reference; the Apple DTS is 616807905.

**Comment by trevorlinton** [comment]
This seems fixed in 10.3-dev release, i'm closing this out as it seems only related to 10.1/10.2 and an apple bug.

Feel free to reopen if it continues to be an issue.
